src/lib/viewers/controls/page/PageControls.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import Browser from '../../../Browser';
+import { BROWSERS } from '../../../constants';
+import { decodeKeydown } from '../../../util';
+import fullscreen from '../../../Fullscreen';
+import IconArrowDown24 from '../icons/IconArrowDown24';
+import IconArrowUp24 from '../icons/IconArrowUp24';
+import './PageControls.scss';
+
+export type Props = {
+    onPageChange: (arg0: number) => void;
+    pageCount: number;
+    pageNumber: number;
+    viewer: HTMLElement;
+};
+
+export default function PageControls({ onPageChange, pageCount, pageNumber, viewer }: Props): JSX.Element {
+    const isSafariFullscreen = Browser.getName() === 'Safari' && fullscreen.isFullscreen();
+    const [isInputShown, setIsInputShown] = React.useState(false);
+    const [tempPageNumber, setTempPageNumber] = React.useState(pageNumber.toString());
```

Comment:
Instead of `tempPageNumber`, could this be something like `pageInputValue`?

Reply:
Sounds good!  Will make that change!